source/2019-11-29-the-ember-times-issue-126.md

```diff
@@ -65,13 +66,17 @@ If you can contribute to everyone's learning today, we ask that you help **updat
 
 ---
 
-## [Section Title in Title Case 🐹](#section-url)
+## [More Improvements Coming to Ember Inspector Soon 🚀](https://github.com/emberjs/ember-inspector/pull/1088)
 
-<change section title emoji>
-<consider adding some bold to your paragraph>
+A huge shoutout and thank you are owed to [Godfrey Chan @chancancode](https://github.com/chancancode) and [Robert Wagner @rwwagner90](https://github.com/rwwagner90), the maintainer of the Ember Inspector, and everyone else involved for all their amazing work in getting the new and improved **Ember Inspector** ready for Octane! 🔥🔥🔥
 
-<add your name to author list, top and bottom>
-<add blurb and emoji to "SOME-INTRO-HERE">
+With the release of Octane coming soon it is welcomed news that Octane's Glimmer components work in Ember Inspector now, in both the component tree and the object inspector. Other improvements include revamping the component hover inspection tool to match Chrome's inspection style, many bug fixes and a complete rewrite of the component tree logic that is more future proof.
```

Comment:
Jared, I think you may have a better idea with the Octane schedule. At a first glance, I wasn't sure if we should say that the release is coming soon. What are your thoughts?

If you'd like, I think including a screenshot or two of the new Inspector in action would be cool here.

Reply:
Yeah that definitely occurred to me that we don't want to imply a particular schedule with Octane. Perhaps I'll just remove the word soon. I think it's pretty uncontroversial that is being released.

Reply:
Good call, I think a screen shot would be great. Or even gif if we can do that.

Reply:
@jaredgalanis just poked around in goodbits. i'm not sure if embeding a gif is possible. we can embed a tweet, that could have a gif (not sure what it would look like tbh, but can try it out)

but we can include an image and link out to a gif, if that's helpful

Reply:
@jaredgalanis would you like me to grab a screenshot?

Reply:
@rwwagner90 that'd be great!

Reply:
also mentioned this on Discord. I'm not sure of the status here. Sorry am a bit less avail with the holiday. FWIW we could always include a .gif / image next week. I know Robert DM'd me mentioning the gif on https://github.com/emberjs/ember-inspector/pull/1088. I'm pretty sure it cannot be captured in Goodbits (and it would be super small) so I'm going to merge this as-is, but I will give the .gif a try in Goodbits